packages/grafbase-sdk/src/federation.ts

```diff
@@ -0,0 +1,15 @@
+export interface FederationParams {
+  version: string
+}
+
+export class Federation {
+  private version: string
```

Comment:
Wonder if it's worth typing this more specifically - we only accept the string `2.3` currently, so I _think_ we could give users faster feedback with the type system?

Reply:
Will add that as a specific value then

Reply:
![Uploading Screenshot 2023-10-11 at 15.04.31.png…]()
Done